beacon/engine/types.go

```diff
@@ -297,8 +297,15 @@ func ExecutableDataToBlockNoHash(data ExecutableData, versionedHashes []common.H
 		withdrawalsRoot = &h
 	}
 
+	isthmusEnabled := bType.IsIsthmus(data.Timestamp)
```

Comment:
Would it be better to use `isPrague` here? And then ensure in the chain config that Prague activates with Isthmus?

Reply:
Ah I see -- yeah that probably makes sense. Apologies for the confusion -- can you paste a link to that part of the spec though (say in the PR description)?

Reply:
Good catch! Looks like that's missing from the spec. Should be part of EIP-7685 though. I'll open a PR on specs for that.

Reply:
Yup, added here: https://github.com/ethereum-optimism/specs/blob/509b58bf8b68049f292ea19686ac9647340a5925/specs/protocol/exec-engine.md#engine_newpayloadv4

Reply:
Thanks. I could be missing something, but I couldn't find "execution requests must be an empty array" in the OP spec?

Reply:
Ah, that's a good point. This references the API call, but not that the block has to include an empty requests hash. This should determine the requests hash of the block, but we should call that out explicitly. I'll open a specs PR for that - issue here: https://github.com/ethereum-optimism/specs/issues/568